index.js

```diff
@@ -117,28 +117,28 @@ var actions = new Actions(config);
   .action((priv_key,data,address,fee,gas_limit,cmd) => actions.transact(priv_key,data,address,fee,gas_limit,cmd.unsafe));
 
   program
-  .command('bond <priv_key> <address> <amount> <fee> <public_key>')
-  .option('-u, --unsafe', 'unsafe sending transaction')
+  .command('bond <address> <amount> <fee> <public_key> <priv_key>')
+  .option('-u, --unsafe', 'unsafe sending transaction') //TODO (unsafe should be implemented using privatekey)
   .alias('bnd')
-  .description('\n(safe) Do Bond transaction, you need pass the private key of sender and address of receiver\
+  .description('\n(safe) Do Bond transaction, you need pass the validator address, stake amount, transaction fee, public key of the validator and private key of sender \
   \nyou may need to initialize a project before using this command.\n\n')
-  .action((priv_key,address,amount,fee,public_key,cmd) => actions.broadcastBond(priv_key,address,parseInt(amount),parseInt(fee),public_key,cmd.unsafe));
+  .action((address,amount,fee,public_key,priv_key) => actions.broadcastBond(address,parseInt(amount),parseInt(fee),public_key,priv_key));
 
   program
-  .command('unbond <priv_key> <address> <amount> <fee>')
+  .command('unbond <address> <amount> <fee> <priv_key>')
   .option('-u, --unsafe', 'unsafe sending transaction')
   .alias('ubnd')
-  .description('\n(safe) Do Unbond transaction, you need pass the private key of sender and address of receiver\
+  .description('\n(safe) Do Unbond transaction, you need pass address of the sender \
   \nyou may need to initialize a project before using this command.\n\n')
-  .action((priv_key,address,amount,fee,cmd) => actions.broadcastUnbond(priv_key,address,parseInt(amount),parseInt(fee),cmd.unsafe));
+  .action((address,amount,fee,priv_key) => actions.broadcastUnbond(address,parseInt(amount),parseInt(fee),priv_key));
 
   program
-  .command('send <priv_key> <address> <amount> ')
-  .option('-u, --unsafe', 'unsafe sending transaction')
+  .command('send <toAddress> <amount> <priv_key>')
```

Comment:
To make this command more compatible with others I suggest that we use `address` instead of `toAddrres` or vice versa.

Reply:
Sure I will implement it

Reply:
modified in commit `ec75375076fba8677df1b3f351223463f5e392e9`